pkgs/data/icons/moka-icon-theme/default.nix

```diff
@@ -1,24 +1,18 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, faba-icon-theme, gtk3 }:
+{ stdenv, fetchFromGitHub, meson, ninja, gtk3, faba-icon-theme }:
 
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
   pname = "moka-icon-theme";
-  version = "5.3.6";
+  version = "5.4.0";
 
   src = fetchFromGitHub {
     owner = "snwh";
     repo = pname;
     rev = "v${version}";
-    sha256 = "17f8k8z8xvib4hkxq0cw9j7bhdpqpv5frrkyc4sbyildcbavzzbr";
+    sha256 = "015l02im4mha5z91dbchxf6xkp66d346bg3xskwg0rh3lglhjsrd";
```

Comment:
ofBorg gets `166wanxa4azqff6h0cw0n98c3gc4zlvjynmigz8v7cq5d4s7jg9h` as a checksum here.

Reply:
It seems that the checksum differ on darwin and linux.